server/src/main/java/de/pnp/manager/model/manager/CharacterHandler.java

```diff
@@ -66,6 +67,10 @@ public void deleteCharacter(String sessionID, String characterID) {
 
         if (characters != null) {
             characters.removeIf(battle -> battle.getCharacterID().equals(characterID));
+            manager.getNetworkHandler().activeBroadcast(
+                    new DismissCharactersMessage(characterID, Calendar.getInstance().getTime()),
+                    manager.getNetworkHandler().clientsProperty().filtered(c -> c.getControlledCharacters().contains(characterID))
```

Comment:
this should also remove the character from the clients's list of controlled characters

Reply:
It does, activeBroadcast uses the sendActiveMessage function, which changes the state of a client